src/js/main.js

```diff
@@ -48,7 +55,36 @@ class Site {
       _this.$covervidVideo.coverVid(1920, 1080);
     }
   }
+
+  submitAgeForm() {
+    $('#submit-age').on('click', function(e){
+      e.preventDefault();
+      const month = $('.month').val();
+      const day = $('.day').val();
+      const year = $('.year').val();
+      const birthday = dayjs(new Date(year, month, day));
+      const age = dayjs().diff(birthday, 'years');
+      if (age >= 21) {
+        Cookies.set('legalAge', true, { expires: 1 }); // Expires in 1 day
+        $('body').addClass('legal-age');
+      } else {
+        console.log("not of age");
+      }
+    });
+  }
+
+  checkForCookie() {
+    const cookie = Cookies.get('legalAge');
+    console.log(cookie);
+    if (cookie) {
+      $('body').addClass('legal-age');
+    } else {
+      console.log('doing nothing')
+    }
+  }
+
+
 }
 
 const Material = new Site();
-const MaterialShop = new Shop();
+//const MaterialShop = new Shop();
```

Comment:
@neener This is a temporary fix for the error you were getting but you should be careful not to commit changes we don't actually need.

Reply:
tip: run `git diff` to review your changes before git add/commit.